File: legacy/src/main/java/br/com/legacy/managers/dtos/Feed.java
package br.com.legacy.managers.dtos;

import java.util.List;

/**
 * Created by appsimples on 1/8/18.
 */

public class Feed {
    public List<Post> post;
    public String lastPostId;
    public String lastMasterPostId;
    public Boolean morePostsToGet;
}

File: feature_feed/src/main/kotlin/br/com/goin/feature/feed/model/Feed.kt
package br.com.goin.feature.feed.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable


data class Feed(@SerializedName("post") val post: List<Post>? = arrayListOf(),
                @SerializedName("lastPostId") val lastPostId: String? = null,
                @SerializedName("lastMasterPostId") val lastMasterPostId: String? = null,
                @SerializedName("morePostsToGet") val morePostsToGet: Boolean = false): Serializable

